src/service/committers.svc.ts

```diff
@@ -0,0 +1,239 @@
+export interface CommitEntry {
+  month: string;
+  author: string;
+}
+
+export interface AuthorCommitCounts {
+  [author: string]: number;
+}
+
+export interface MonthlyData {
+  [month: string]: AuthorCommitCounts;
+}
+
+export interface CommitEntry {
+  month: string;
+  author: string;
+}
+
+export interface ReportData {
+  monthly: {
+    [month: string]: {
+      [author: string]: number;
+      total: number;
+    };
+  };
+  overall: {
+    [author: string]: number;
+    total: number;
+  };
+}
+
+/**
+ * Parses git log output into structured data
+ * @param output - Git log command output
+ * @returns Parsed commit entries
+ */
+export function parseGitLogOutput(output: string): CommitEntry[] {
+  return output
+    .split('\n')
+    .filter(Boolean)
```

Comment:
```suggestion
    .filter(Boolean)
```

I assume we just want to filter empty strings? so we can just filter those with coercing it to boolean

Reply:
Filtering out empty strings is the main goal, but using .filter(Boolean) also protects us in case a section between two `\n` somehow resolves to null or undefined.